flutter-ory-network/lib/main.dart

```diff
@@ -36,7 +38,18 @@ Future<void> main() async {
   final dio = DioForNative(options);
 
   final authService = AuthService(dio);
-  final authRepository = AuthRepository(service: authService);
+
+  final googleSignIn = GoogleSignIn(
+      clientId: Platform.isAndroid
+          ? dotenv.get('WEB_CLIENT_ID')
+          : dotenv.get('IOS_CLIENT_ID'),
```

Comment:
Could you add a comment why this is called `"WEB_CLIENT_ID"`? That way, we don't need to ask ourselves that in a couple of months.